Directory.Build.props

```diff
@@ -3,7 +3,10 @@
 	<PropertyGroup>
 		<LangVersion>latest</LangVersion>
 		<Nullable>enable</Nullable>
-		<WarningsAsErrors>nullable,CS1574</WarningsAsErrors><!--CS1574: XML comment has cref attribute that could not be resolved-->
+		<!--CS1574: XML comment has cref attribute that could not be resolved-->
+ 		<!--CS1573: Parameter has no matching param tag in the XML comment -->
+ 		<!--CS1734: XML comment has a paramref tag, but there is no parameter by that name-->
+ 		<WarningsAsErrors>nullable,CS1574,CS1573,CS1734</WarningsAsErrors>
```

Comment:
can we enable all warnings as errors? but use Exclude with NoWarn

Reply:
Interesting thought! 

I'm not against it. I'm trying to think of scenarios where this could cause problems, but it may be something that we may need to try it first.